static/app/components/devtoolbar/components/releases/sessionStatusBadge.tsx

```diff
@@ -0,0 +1,24 @@
+import IndicatorBadge from 'sentry/components/devtoolbar/components/indicatorBadge';
+import useSessionStatus from 'sentry/components/devtoolbar/components/releases/useSessionStatus';
+import {IconFire} from 'sentry/icons';
+
+export default function CrashCountBadge() {
+  const {data} = useSessionStatus();
+  const healthySessions = data?.json.groups.find(
+    g => g.by['session.status'] === 'healthy'
+  )?.totals['sum(session)'];
+  const crashSessions = data?.json.groups.find(g => g.by['session.status'] === 'crashed')
+    ?.totals['sum(session)'];
+
+  if (!crashSessions || !healthySessions) {
+    return null;
+  }
+
+  const crashPercent = (crashSessions / healthySessions) * 100;
```

Comment:
Since it's percentage of sessions that were crashes, should we be using total sessions instead?
```suggestion
  const crashPercent = (crashSessions / (crashSessions + healthySessions)) * 100;
```

Reply:
oh yes good catch